### PR TITLE
Add logging to the front end

### PR DESCRIPTION
**Pull request by irisfaraway**
https://eaflood.atlassian.net/browse/WE-615

Add logging so we can understand what the app is doing, identify unexpected behaviour and diagnose issues.

Still a WIP right now.

### CODE REVIEW

**Comment by irisfaraway** [comment]
As [Good is only intended to log Hapi events](https://github.com/hapijs/good/issues/308), we are leaving our `console.log`s and `console.error`s in their current format for now.

Requests for static assets no longer appear in the log though.

File: latestCommit.json [hunk 1]
```diff
@@ -0,0 +1 @@
+f7d2e924bdff99886feddd529c922b6b99285ec0
```
Comment (duncannixon) [inline latestCommit.json]: This file no longer exists as we are now saving the latest commit to an environment variable during the gulp task instead.

**Comment by irisfaraway** [comment]
@Cruikshanks @duncannixon Think I've addressed all the issues from the review – would you mind having another look? PR looks massive now but most of the files have very minor changes only.

**Comment by irisfaraway** [comment]
@Cruikshanks 😭  All good points. I'll sort this out today.

**Comment by irisfaraway** [comment]
@Cruikshanks Think I've covered your comments now – would you mind having another look?

**Pull request merged**